tests/extensibleLM/docker-compose.yml

```diff
@@ -0,0 +1,154 @@
+version: '3'
+networks:
+  pulsar:
+    driver: bridge
+services:
+  # Start ZooKeeper
+  zookeeper:
```

Comment:
@BewareMyPower do you happen to know why this zk server cannot start? It is a bit tricky to debug as not outputting the logs.

Reply:
It's a little weird. I think you can test this startup script separately in GitHub Actions.

Reply:
For example, it can be reproduced like: https://github.com/BewareMyPower/pulsar-client-cpp/actions/runs/7620840669/job/20756233457?pr=28

Reply:
I might find the cause, see:
- https://github.com/BewareMyPower/pulsar-client-cpp/pull/28/commits/e530edef9a07ada654e35d09a83f02b5de518965
- https://github.com/BewareMyPower/pulsar-client-cpp/actions/runs/7621067202/job/20756849641?pr=28

Removing the volume mappings works.